### PR TITLE
Removing unused imports, cells and __future__ from the notebooks.

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
The review notebook app is really good! I never knew it existed!

File: docs/tutorials/_template.ipynb [hunk 1]
```diff
@@ -109,19 +109,6 @@
     "[Put all your imports and installs up into a setup section.]"
```
Comment (WindQAQ) [inline docs/tutorials/_template.ipynb]: <p>Shouldn't this be <code>~=0.7</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

**Comment by WindQAQ** [comment]
Nice cleanup! It seems that you are using pycharm to edit notebook so that the indentation of raw notebook file changes a lot. Could we use colab instead, as our examples/tutorials are ran on it?

**Comment by gabrieldemarmiesse** [comment]
Sure thing, I'll do that. I'm not used to use notebooks with git.

File: docs/tutorials/_template.ipynb (hunk 1 above)
Comment (gabrieldemarmiesse) [reply docs/tutorials/_template.ipynb]: My bad. Let me change that.

**Comment by gabrieldemarmiesse** [comment]
@WindQAQ  I started again to modify the notebook from scratch with colab and the diff still looks horrible. Is there a way to modify the notebook in a diff friendly way?

**Pull request closed without merging**